### ISSUE TITLE
Difference to Redshift

### ISSUE THREAD

**Issue by thomas-mc-work** [jumper149/blugon#7]
I'm wondering what bluegon can do better/different from [Redshift](https://github.com/jonls/redshift/)? Maybe you should also add that to the README.md.

**Comment by jumper149** [comment]
I myself am not too familiar with redshift and also f.lux which seem to be the most popular competitors to blugon. One major reasoned I created blugon for, was the ability to control the gamma values on particular daytimes instead of the way redshift does it which is only night and day.

Blugon is also a lot simpler and you can even say less sophisticated.

I like the idea of a comparison table for the readme, but would like to hear from other people what their reason for using blugon was!

**Comment by jumper149** [comment]
I would really like someone to look at #8. I am not sure with all the comparisons! Are there other categories that are missing?

**Comment by temporaryrespite** [comment]
From my limited interaction with Redshift (thank you for suggesting it btw), all I know is that it can apply color temperature **relative** to the current color temperature, such as:
 `redshift -O 3000` (for 3000K)
as opposed to absolute: `redshift -P -O 3000` which would act like `blugon -r -o -b scg` if `/home/user/.config/blugon/current` contains the value `3000.0`.

>   -O TEMP	One shot manual mode (set color temperature)
>   -P		Reset existing gamma ramps before applying new color effect

for tty/drm you'd have to append `-m drm` or else `-m randr` is assumed!

>   -m METHOD	Method to use to set color temperature
  		(Type `list' to see available methods)

```
$ redshift -m list
Available adjustment methods:
  drm
  randr
  vidmode
  dummy

Specify colon-separated options with `-m METHOD:OPTIONS'.
Try `-m METHOD:help' for help.
```

I initially thought it was a [bug](https://github.com/jonls/redshift/issues/395#issuecomment-576452306), but I can see its usefulness as a feature now.

My only usage of blugon thus far was to set color temperature, for both tty/drm and for X. So I made [this](https://github.com/temporaryrespite/secote) blugon-inspired toy(to do just that) before I knew of Redshift.

**Comment by jumper149** [comment]
fixed by #8.
